src/ORM/FieldType/DBClassName.php

```diff
@@ -20,8 +22,13 @@ public function getFieldSpec(): string|array
         return $spec;
     }
 
+    /**
+     * @inheritDoc
+     * @deprecated 6.2.0 Use getDefaultValue() instead.
+     */
     public function getDefault(): string
     {
+        Deprecation::notice('6.2.0', 'Use getDefaultValue() instead.');
```

Comment:
Why are we getting rid of this method?

Reply:
Because we should be using `getDefaultValue()`. We don't need two methods to get the default value - and we especially don't want them to provide different results.